Gem/Transfer/Types/PriceImpact.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+import Foundation
+import Style
+import SwiftUICore
+
+enum PriceImpactType: Equatable {
+    case none
+    case low(String)
+    case medium(String)
+    case high(String)
+    case positive(String)
+    
+    var value: String? {
+        switch self {
+        case .low:
+            return nil // Decided not to show low price impact.
+        case .medium(let value):
+            return value
```

Comment:
```suggestion
        case .medium(let value): value
```

don't use return, unless it's needed